flight/targets/coptercontrol/fw/pios_board.c

```diff
@@ -856,11 +856,13 @@ void PIOS_Board_Init(void) {
 #endif	/* PIOS_INCLUDE_PWM */
 		break;
 	case HWCOPTERCONTROL_RCVRPORT_PPM:
```

Comment:
want to call this PPMONPIN1 for consistency? maybe not worth breaking .uav files.